gad/Detector/mod_util.py

```diff
@@ -23,12 +24,18 @@ def find_seg(flag):
 def plot_seg(X, Y, flag, marker=None, *args, **kwargs):
     """plot X and Y, but the points with flag == true to be one color, the result to be another color
     """
+    font = {'family': 'normal',
+            'weight': 'normal',
+            'size': 16}
+
+    matplotlib.rc('font', **font)
+
     segs = find_seg(flag)
     for a, b, tp in segs:
         if marker[tp] is not None:
             # import ipdb;ipdb.set_trace()
             # plt.plot(X[a:b], Y[a:b], marker[tp], *args, **kwargs)
-            plt.plot(np.array(X[a:b]), np.array(Y[a:b]), marker[tp], *args, **kwargs)
+            plt.plot(np.array(X[a:b]), np.array(Y[a:b]), marker[tp], linewidth=2, *args, **kwargs)
```

Comment:
could you add a parameter to plot_seg for linewidth instead of hardcoding here? 2 can be the default value